gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinHelixTask.java

```diff
@@ -194,6 +195,7 @@ private ContainerHealthCheckFailureEvent createTaskCreationEvent() {
     event.addMetadata("helixJobId", this.helixJobId);
```

Comment:
Can we align the metadata field names to the ones in HelixTaskEventMetadataGenerator?